### DISCUSSION TITLE
Plan to add multiple tab panels plugin?

### DISCUSSION THREAD

**Discussion by simonwei97** [alshedivat/al-folio#1977] [Ideas]
Will have plan to add this plugin https://github.com/Ovski4/jekyll-tabs for multi tab panels on the same page.

## Demo

https://github.com/Ovski4/jekyll-tabs/raw/master/docs/tabs-example.png

**Comment by george-gca** [comment]
It would be a great addition. We are accepting PRs if you are willing to do it.

**Comment by george-gca** [comment]
Implemented in #2380.
